website/content/partials/known-issues/internal-error-namespace-missing-policy.mdx

```diff
@@ -9,4 +9,53 @@ This impacts all auth methods.
 - 1.14.4 and 1.14.5
 - 1.15.0 and 1.15.1
 
-A fix will be released in Vault 1.15.2, 1.14.6, and 1.13.10.
\ No newline at end of file
+A fix will be released in Vault 1.15.2, 1.14.6, and 1.13.10.
+
+### Workaround
+During authentication, Vault will derive any additional policies that should be inherited based on any groups an entity is a member of. If Vault attempts to attach a policy to a token from a different namespace than the one handling the authentication, _and_ that policy does not exist, Vault will return an internal error. The way to resolve this issue is to either add the policy, or to delete the group policy mapping referring to the non-existent policy.
+
+Below is an example of a userpass auth method that is failing because of a
+missing group policy from another namespace.
```

Comment:
```suggestion
As an example, consider the following userpass auth method failure. The error is
likely due to the fact that Vault expects a group policy under the namespace for
`user1` that does not exist.
```
Style correction: write in active voice, make all participating elements explicit